views/js/qtiCreator/widgets/item/Widget.js

```diff
@@ -75,6 +75,8 @@ define([
 
         this.itemUri = config.uri;
 
+        xmlRenderer.switchPerInteractionRP(config.perInteractionRp);
```

Comment:
in the `item/Widget` will be changed the state of the renderer to per interaction response processing. but it will be used in this state in other places as well. the state changed in this file because it is entry point of test item initialization